scripts/build_framework.sh

```diff
@@ -120,7 +120,6 @@ function xcode_build_target() {
     -configuration "${2}" \
     SYMROOT="$BOLTS_BUILD" \
     CURRENT_PROJECT_VERSION="$BOLTS_VERSION_FULL" \
-    clean build \
```

Comment:
Can we do just `build` without `clean`?
This will clarify what you are trying to do?

Reply:
hi @nlutsenko 

this changes where discussed in facebook/facebook-sdk-swift#246,
i just implemented it on this pr on request from @codytwinton 

i can test on my side to see if using just `build` would work

Reply:
I guess, `xcodebuild clean` internally performs `rm` but seems like the new build system checks if the directory was created in the process. In this case `build` directory is created by `mkdir` manually without `xcodebuild` and end up with failing.